tests/tracer/test_span.py

```diff
@@ -676,3 +676,14 @@ def test_manual_context_usage():
     span1.context.sampling_priority = 1
     assert span2.context.sampling_priority == 1
     assert span1.context.sampling_priority == 1
+
+
+def test_set_exc_info_with_unicode():
+    try:
+        raise Exception("DataDog/水")
```

Comment:
would it make sense to test both utf8 encoded `str` and `unicode` for python2?
```python
if six.PY3:
    Exception("DataDog/水")
else:
    Exception("DataDog/水")
    Exception(u"DataDog/水")
```

Reply:
did that in https://github.com/DataDog/dd-trace-py/pull/3016/commits/c9bc517dbd2b005a9fe7e273a8a4e378b54421f7

Reply:
update here, we are no longer test utf-8 `str` for Python 2 since we cannot figure out how to actually get the `Exception` -> `str` properly without raising an exception.

We figured we can at least make sure Python 3 + `unicode` in Python 2 work well, then we can handle the other as a follow-up if need be.